clue.js

```diff
@@ -0,0 +1,282 @@
+  }
+
+  const colMustard = {
+    firstName: 'Jack',
+    lastName: 'Mustard',
+    color: 'yellow',
+    description: 'Great white hunter and colonial imperialist',
+    age: 55,
+    image: 'media/mustard.png',
+    occupation: 'Colonel'
+  }
+  const mrsWhite = {
+    firstName: 'Mrs',
+    lastName: 'White',
+    color: 'lightgray',
+    description: 'She used to practice medicine and knows how to clean up blood',
+    age: 60,
+    image: 'media/white.png',
+    occupation: 'Housekeeper'
+  }
+
+  const suspects = [mrGreen, profPlum, msScarlet, mrsPeacock, colMustard, mrsWhite]
+  return suspects
+}
+// THe weapons
+function weaponList() {
+  const rope = {
+    name: 'rope',
+    weight: 10,
+    color: 'antiquewhite',
+    description: 'Old but sturdy the favorite of Plum'
+  }
+  const knife = {
+    name: 'knife',
+    weight: 15,
+    color: 'blue',
+    description: 'Mrs Peacock´s favorite weapon and always sharp'
+  }
+  const candlestick = {
+    name: 'candlestick',
+    weight: 50,
+    color: 'gold',
+    description: 'An old heirloom from the Peacock family'
+  }
+  const dumbbell = {
+    name: 'dumbbell',
+    weight: 100,
+    color: 'gray',
+    description: 'Heavy but very deadly'
+  }
+  const poison = {
+    name: 'poison',
+    weight: 5,
+    color: 'green',
+    description: 'Cynonide, usually a womans weapon'
+  }
+  const axe = {
+    name: 'axe',
+    weight: 40,
+    color: 'red',
+    description: 'In case of a fire or a heated argument'
+  }
+  const bat = {
+    name: 'bat',
+    weight: 30,
+    color: 'brown',
+    description: 'Mr Green´s favorite bat'
+  }
+  const trophy = {
+    name: 'trophy',
+    weight: 45,
+    color: 'silver',
+    description: 'Best hunter this season, awarded to Colonel Mustard'
+  }
+  const pistol = {
+    name: 'pistol',
+    weight: 20,
+    color: 'darkslategray',
+    description: 'small but effective'
+  }
+
+  const weapons = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol]
+  return weapons
+
+  // The different times
+}
+function timeList() {
+  const morning = {
+    name: '8:00',
+    color: 'wheat'
+  }
+  const noon = {
+    name: '12:00',
+    color: 'skyblue'
+  }
+  const afternoon = {
+    name: '17:00',
+    color: 'pink'
+  }
+  const evening = {
+    name: '21:00',
+    color: 'indigo'
+  }
+  const night = {
+    name: '1:00',
+    color: 'black'
+  }
+  const times = [morning, noon, afternoon, evening, night]
+  return times
+}
+// The different rooms
+
+function roomList() {
+  const rooms = ["dining room", "conservatory", "kitchen", "study", "library", "billiard room", "lounge", "ballroom", "hall", "spa", "living room", "observatory", "theater", "guest house", "patio"]
+  return rooms
+}
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)]
+}
+
+let mystery = {
+  killer: "blank",
```

Comment:
You can have them as empty!
```suggestion
  killer: "",
```